src/components/ItemEditorPage/LeftPanel/Toasts/ItemAdded/ItemAddedToast.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from 'react'
+import { Button, Toast, ToastType } from 'decentraland-ui'
+import { Link, useHistory, useLocation } from 'react-router-dom'
+import { locations } from 'routing/locations'
+import { T, t } from 'decentraland-dapps/dist/modules/translation/utils'
+
+import ItemAddedLogo from '../../../../../icons/check.svg'
+
+import { Props } from './ItemAddedToast.types'
+import './ItemAddedToast.css'
+
+const ItemAddedToast: React.FC<Props> = props => {
+  const { collectionId } = props
+  const [shouldShowToast, setShouldShowToast] = useState(false)
+  const [item, setItem] = useState<string | undefined>()
+  const search = useLocation().search
+  const history = useHistory()
+
+  useEffect(() => {
+    const searchParams = new URLSearchParams(search)
+    const newItem = searchParams.get('newItem')
```

Comment:
could you please move this get `newItem` query param to `modules.locations.selectors` please? like this one: https://github.com/decentraland/builder/blob/master/src/modules/location/selectors.ts#L54